### PR TITLE
Splitting the tests in test_augmentation_3d.py and test_container.py into individual test files.

### LINKED ISSUE #2750: [Tracker] Split tests into multiple file tests
Currently we have some test files which have all (or almost all) tests of a module within a single file, the ideal and goal here is to split these cases into multiple test files. Which file should be testing a single functionality, or file of our library.

Example:
We have now a single file testing almost all losses functions https://github.com/kornia/kornia/blob/c5558ac2c03bbbbbbb6c9f82616b8d5229398f1a/tests/losses/test_losses.py we would like to have a single test file for each loss. What would mirror the file organization of the losses module of kornia https://github.com/kornia/kornia/tree/c5558ac2c03bbbbbbb6c9f82616b8d5229398f1a/kornia/losses

This issue will work as a tracker to map some of these files which need to be split, feel free to help us find others and/or help us splitting them into multiple files.

The goal here is to reorganize our test suite. If you have any questions, you can call us and/or ask on our Slack page too [Slack](https://join.slack.com/t/kornia/shared_invite/zt-csobk21g-2AQRi~X9Uu6PLMuUZdvfjA)

_Originally posted by @edgarriba in https://github.com/kornia/kornia/pull/2745#discussion_r1461605306_

### PR DESCRIPTION
**Pull request by AbdulrahmanMahmoudRiyad**
#### Changes

A. As requested, I split container augmentation tests (https://github.com/kornia/kornia/blob/ce434e467faf617604bb3383cf78cd0b79f59dbd/tests/augmentation/test_container.py) into the following files:

1. utils.py (reproducibility_test) (placed in testing/augmentation)
2. test_video_sequential.py
3. test_sequential_aug.py
4. test_augmentation_sequential.py
5. test_patch_sequential.py
6. test_dispatcher.py

B. Also, I split augmentations tests (to be specific the 3d not both 2d and 3d) (https://github.com/kornia/kornia/blob/ce434e467faf617604bb3383cf78cd0b79f59dbd/tests/augmentation/test_augmentation_3d.py) into the following files:

1. test_random_horizontal_flip_3d.py
2. test_random_vertical_flip_3d.py
3. test_random_depthical_flip_3d.py
4. test_random_rotation_3d.py
5. test_random_crop_3d.py
6. test_center_crop_3d.py
7. test_random_equalize_3d.py
8. test_random_affine_3d.py

Fixes #2750 (Splits test_augmentation_3d.py into multiple test files and Splits test_container.py into multiple test files)

#### Type of change
Improved modularity for test files.

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by AbdulrahmanMahmoudRiyad** [comment]
Good morning,

This pull request splits the augmentation tests from `test_augmentation_3d.py` and `test_container.py` into individual test files for better organization. All tests have been verified to pass using `pytest`.

This is part of issue #2750.

@edgarriba @johnnv1 Please review this pull request. Any feedback is very welcomed

**Review by edgarriba — COMMENTED** [review]
are this new tests or you forgot to remove the originals ?

**Comment by AbdulrahmanMahmoudRiyad** [comment]
> are this new tests or you forgot to remove the originals ?

Yes, I forgot to remove the originals.

**Comment by AbdulrahmanMahmoudRiyad** [comment]
I would like to request some guidance to fix the failed checks.

**Comment by AbdulrahmanMahmoudRiyad** [comment]
I think I have done it.

File: tests/augmentation/reproducibility_test.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import torch
+
+from testing.base import assert_close
+
+
+def reproducibility_test(input, seq):
```
Comment (johnnv1) [inline tests/augmentation/reproducibility_test.py:23]: This function should go to testing/augmentation. It's a utility function for tests, not a test itself

**Review by johnnv1 — CHANGES_REQUESTED** [review]
Thanks for raising this PR. could you move the test files to kinda follow up the structure under kornia/augmentation? This way we should have subdirectories for each group of tests, like 2d,3d,containers,auto, etc etc

File: tests/augmentation/reproducibility_test.py (hunk 1 above)
Comment (AbdulrahmanMahmoudRiyad) [reply tests/augmentation/reproducibility_test.py]: Thank you; I really appreciate the feedback. I organized augmentation tests into the subdirectories _3d and container as you said. I also moved the utility function (which got renamed from reproducability_test.py to utils.py) to testing/augmentation.

**Comment by AbdulrahmanMahmoudRiyad** [comment]
@edgarriba @johnnv1 Any feedback is more than welcome. There are just 2 checks failing and I don't think they are related to what I did at all. Can you help me understand them?

**Review by johnnv1 — APPROVED** [review]
Thanks for the patch
